### ISSUE TITLE
version 4.0 is available only from pip3

### ISSUE THREAD

**Issue by f18m** [bemeurer/beautysh#45]
Hi @bemeurer ,
I noticed that I can install the new release (4.0) only by doing:
   pip3 install beautysh
if I do: 
   pip install beautysh==4.0
I get:

```
Collecting beautysh==4.0
  Could not find a version that satisfies the requirement beautysh==4.0 (from versions: 4.0.macosx-10.14-x86_64, 2.2.2, 3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6, 3.7, 3.8, 3.9, 3.10, 3.11, 3.12)
No matching distribution found for beautysh==4.0
```

I think this is due to the fact that on pypi there is no .tar.gz...

**Comment by bemeurer** [comment]
Fixed in 4.1, thanks for the report!

**Comment by f18m** [comment]
verified that it's fixed in 4.1, thanks!!
